src/class-wpml-gutenberg-strings-registration.php

```diff
@@ -52,24 +52,29 @@ public function register_strings( WP_Post $post, $package_data ) {
 		$this->leftover_strings = $original_strings = $this->string_translation->get_package_strings( $package_data );
 		$this->string_location  = 1;
 
+		$stringCleanUp = apply_filters( 'wpml_pb_get_string_clean_up', '', $post->ID );
+
 		$this->register_blocks(
 			WPML_Gutenberg_Integration::parse_blocks( $post->post_content ),
 			$package_data,
-			$post->ID
+			$post->ID,
+			$stringCleanUp
 		);
 
 		$current_strings = $this->string_translation->get_package_strings( $package_data );
 
 		$this->reuse_translations->find_and_reuse_translations( $original_strings, $current_strings, $this->leftover_strings );
 
+		$stringCleanUp->cleanUp();
```

Comment:
`$stringCleanUp` can be an empty string, so you should check it before to call the method.

Reply:
@strategio I've refactored this.